### PR TITLE
Translated in italian the "Container Lifecycle Hook" chapter

### LINKED ISSUE #20962: Localize "Container Lifecycle Hooks" page to Italian
I would like to have the "Container Lifecycle Hooks" page to be translated into Italian
page https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/

/language it
/priority important-soon

### PR DESCRIPTION
**Pull request by davideimola**
Hi! This is the Italian translation for the "Container Lifecycle Hooks" chapter of the documentation!
Fix #20962

### CODE REVIEW

**Comment by davideimola** [comment]
Related to issue #20962

File: content/it/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+come il salvataggio dello stato del container prima della sua fine.
+
+### Garanzia della consegna dell'hook
+
+La consegna dell'hook è intesa per essere *almeno una volta*, il che significa
+che un hook può essere chiamato più volte da un dato evento, come per `PostStart`
+o `PreStop`.
+Sta all'implementazione dell'hook gestire correttamente questo aspetto.
+
+Generalmente, vengono effettuate solo consegne singole.
```
Comment (fabriziopandini) [inline content/it/docs/concepts/containers/container-lifecycle-hooks.md:92]:
```suggestion
Generalmente, vengono effettuate singole chiamate agli hook.
```

File: content/it/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+Se, per esempio, un ricevitore di hook HTTP è giù e non è in grado di ricevere traffico,
```
Comment (fabriziopandini) [inline content/it/docs/concepts/containers/container-lifecycle-hooks.md:93]:
```suggestion
Se, per esempio, la destinazione di hook HTTP non momentaneamente in grado di ricevere traffico,
```

File: content/it/docs/concepts/containers/container-lifecycle-hooks.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+non c'è alcun tentativo di re invio.
+In alcuni rari casi, tuttavia, può verificarsi una doppia consegna.
```
Comment (fabriziopandini) [inline content/it/docs/concepts/containers/container-lifecycle-hooks.md:95]:
```suggestion
In alcuni rari casi, tuttavia, può verificarsi una doppia chiamata.
```

File: content/it/docs/concepts/containers/container-lifecycle-hooks.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+Per esempio, se un kubelet si riavvia nel mentre dell'invio di un hook, questo potrebbe essere
+re inviato dopo che il kubelet è tornato in funzione.
```
Comment (fabriziopandini) [inline content/it/docs/concepts/containers/container-lifecycle-hooks.md:97]:
```suggestion
chiamato per una seconda volta dopo che il kubelet è tornato in funzione.
```

**Review by fabriziopandini — COMMENTED** [review]
@davideimola welcome to the Kubernetes web site!
The PR looks great! only few nit, but overall lgtm

**Comment by davideimola** [comment]
> @davideimola welcome to the Kubernetes web site!
> The PR looks great! only few nit, but overall lgtm

@fabriziopandini thank you for the warm welcome and for all the suggestions!!

**Comment by fabriziopandini** [comment]
/lgtm
@Fale @micheleberardi @mattiaperi PTAL

**Review by Fale — COMMENTED** [review]
There is a very minor thing, for the rest lgtm :-)

**Comment by davideimola** [comment]
@Fale thanks for the hint! I have applied your suggestion!

**Comment by Fale** [comment]
Thanks @davideimola!
/lgtm

**Comment by fabriziopandini** [comment]
@davideimola please squash commits

**Comment by davideimola** [comment]
@fabriziopandini done

**Comment by fabriziopandini** [comment]
/lgtm
/approve
🥳

**Pull request merged**